HPXMLtoOpenStudio/measure.rb

```diff
@@ -1268,6 +1268,8 @@ def self.create_or_get_space(model, spaces, spacetype)
 
   def self.add_roofs(runner, model, spaces)
     @hpxml.roofs.each do |roof|
+      next if roof.net_area < 0.1 # skip surfaces smaller than 0.1 sqft
```

Comment:
So I'm seeing checks like this for only 3 surface types: roofs, walls, and foundation_walls. I assume those three are called out because they are the only surfaces that can have subsurfaces, so we're checking the net area here (while the HPXML class already checked gross area).

If that's the logic, then I suggest we change the comments here and below from:
`# skip surfaces smaller than 0.1 sqft`
to:
`# skip modeling net surface area for surfaces comprised entirely of subsurface area`